WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductInOrder.swift

```diff
@@ -20,11 +24,26 @@ struct ProductInOrder: View {
                         ProductRow(viewModel: viewModel.productRowViewModel)
                             .padding()
                         Divider()
+                        VStack(spacing: Layout.noSpacing) {
+                            Button(Localization.addDiscount) {
+                                shouldShowDiscountLineDetails = true
+                            }
+                                .buttonStyle(PlusButtonStyle())
+                                .padding()
+                                .accessibilityIdentifier("add-discount-button")
+                            Divider()
+                        }
+                        .renderedIf(viewModel.isAddingDiscountToProductEnabled)
                     }
                     .background(Color(.listForeground(modal: false)))
-
+                    .sheet(isPresented: $shouldShowDiscountLineDetails) {
+                        FeeOrDiscountLineDetails(viewModel: FeeOrDiscountLineDetailsViewModel(isExistingLine: false,
+                                                                                              baseAmountForPercentage: 50,
+                                                                                              total: "0.00",
+                                                                                              lineType: .discount,
+                                                                                              didSelectSave: { _ in }))
+                    }
```

Comment:
Should we extract this initial setup to some sort of static helper? For example:
```suggestion
FeeOrDiscountLineDetails(viewModel: FeeOrDiscountLineDetailsViewModel.createInitial())
```
Then:
```
class FeeOrDiscountLineDetailsViewModel: ObservableObject {
    
    static func createInitial() -> FeeOrDiscountLineDetailsViewModel {
        return FeeOrDiscountLineDetailsViewModel(isExistingLine: false,
                                                 baseAmountForPercentage: 50,
                                                 total: "0.00",
                                                 lineType: .discount,
                                                 didSelectSave: { _ in })
    }
... 
```

Reply:
This will be done in the next PR, when we implement the business logic.